src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the start of the class definition.